Fixed setup for key pairs and GitHub (#294)
carpentries-incubator/python-intermediate-development: setup.md

~~~
{: .language-bash}
~~~
Generating public/private ed25519 key pair.

> Enter file in which to save the key (/Users/user/.ssh/id_ed25519): 

Reviewer: ```suggestion
Enter file in which to save the key (/Users/<YOUR_USERNAME>/.ssh/id_ed25519): 
```

Enter passphrase (empty for no passphrase): 
Enter same passphrase again: 

> Your identification has been saved in /Users/user/.ssh/id_ed25519
> Your public key has been saved in /Users/user/.ssh/id_ed25519.pub

Reviewer: ```suggestion
Your identification has been saved in /Users/<YOUR_USERNAME>/.ssh/id_ed25519
Your public key has been saved in /Users/<YOUR_USERNAME>/.ssh/id_ed25519.pub
```

The key fingerprint is:
SHA256:qjhN/iO42nnYmlpink2UTzaJpP8084yx6L2iQkVKdHk your-github-email@example.com
The key's randomart image is:
+--[ED25519 256]--+
|.. ..            |
| ..o A           |
|. o..            |
| .o.o .          |
| ..+ =  B        |
| .o = ..         |
|o..X *.          |
|++B=@.X          |
|+*XOoOo+         |
+----[SHA256]-----+
~~~
{: .output}

Next, you need to copy your public key (**not your private key - this is important!**) over to 
your GitHub account. The `ssh-keygen` command above will let you know where your public key is saved (the file should have the 

> extension ".pub"), and you can get its contents as follows (on a Mac OS system):

Reviewer: ```suggestion
extension ".pub"). You can get its contents as follows (this is on a Mac OS system, you should change the filepath to the location where your public key is saved):
```

Author: ```suggestion
extension ".pub"), and you can get its contents (e.g. on a Mac OS system) as follows:
```
